### DISCUSSION TITLE
Support for browsers other than chrome

### DISCUSSION THREAD

**Discussion by jithendrauppalapati** [JamesBrill/react-speech-recognition#235] [General]
Hello team,

We integrated this package into our project and it is working as expected in Chrome, But we are unable to implement this in other browsers and platforms. Please guide us to implement this in other browsers, As this is one of our critical business requirements for our app. Waiting to hear from you soon.

**Comment by YoussefZidan** [comment]
> Hello team,
> 
> We integrated this package into our project and it is working as expected in Chrome, But we are unable to implement this in other browsers and platforms. Please guide us to implement this in other browsers, As this is one of our critical business requirements for our app. Waiting to hear from you soon.

Check this section: https://github.com/JamesBrill/react-speech-recognition#cross-browser-example
Also check this issue: https://github.com/JamesBrill/react-speech-recognition/issues/145
